stl/inc/functional

```diff
@@ -73,25 +62,18 @@ struct negate {
     }
 };
 
-// STRUCT TEMPLATE equal_to
 // defined in <xstddef>
 
-// STRUCT TEMPLATE not_equal_to
 // defined in <xstddef>
 
-// STRUCT TEMPLATE greater
 // defined in <xstddef>
 
-// STRUCT TEMPLATE less
 // defined in <xstddef>
 
-// STRUCT TEMPLATE greater_equal
 // defined in <xstddef>
 
-// STRUCT TEMPLATE less_equal
 // defined in <xstddef>
 

```

Comment:
more to  fix

Reply:
and below in this file

Reply:
I'll do an audit of `// defined in <>` and replace them with `x, y, z defined in <a>`. Thanks for pointing this out.